### ISSUE TITLE
Groups UI: Pressing Enter in empty input field deletes group without warning

### ISSUE THREAD

**Issue by aseeland** [IQSS/dataverse#5677]
On the "manage-groups"-page we experienced the following problem: When editing an existing group, placing the cursor in the input field for Group Members, and then (accidentally) pressing "Enter", the whole group is deleted without warning dialog :-(

The same happens in the "create new group"-dialog.

We have Dataverse Version 4.9.4, but I could reproduce the problem on demo.dataverse.org which runs Version 4.11

**Comment by pdurbin** [comment]
😞

**Comment by sekmiller** [comment]
The "enter key" v. "save button" was not a red herring. Somehow, the enter key was invoking the "Continue" button on the Delete Confirmation popup. Previously this popup is rendered, but not visible until you press the trash can button. I added code that leaves the popup unrendered until you press the trash can button.  That fixes the only issue I was able to recreate, that is in Firefox when you press "enter" with the cursor in the autocomplete role assignee control the group is deleted.
